checkstyle/import-control.xml

```diff
@@ -157,6 +157,8 @@
     <allow pkg="org.apache.kafka.connect.errors" />
     <allow pkg="org.apache.kafka.clients" />
     <allow pkg="org.apache.kafka.test"/>
+    <allow pkg="org.reflections"/>
```

Comment:
Should these be further down within `connect.runtime`? I don't think we should need `reflections` anywhere outside the runtime package, right?